src/controller/eme-controller.js

```diff
@@ -63,6 +63,26 @@ const createWidevineMediaKeySystemConfigurations = function (audioCodecs, videoC
   ];
 };
 
+const createPlayreadyMediaKeySystemConfigurations = function (audioCodecs, videoCodecs) { /* jshint ignore:line */
+  const baseConfig = {
+    // initDataTypes: ['keyids', 'mp4'],
+    // label: "",
+    // persistentState: "not-allowed", // or "required" ?
+    // distinctiveIdentifier: "not-allowed", // or "required" ?
+    // sessionTypes: ['temporary'],
+    audioCapabilities: [
+      // { contentType: 'audio/mp4; codecs="mp4a.40.2"' }
+    ],
+    videoCapabilities: [
+      // { contentType: 'video/mp4; codecs="avc1.42E01E"' }
+    ]
+  };
+  // TODO add proper video and audio codecs for playready
```

Comment:
should we do it right now?

Reply:
Sorry didnt see this. We can do this later.